packages/ariakit-playground/src/playground-editor.tsx

```diff
@@ -292,6 +269,29 @@ export const usePlaygroundEditor = createHook<PlaygroundEditorOptions>(
       highlight: !editorDOM && props.highlight,
     });
 
+    props = useWrapElement(
+      props,
+      (element) => (
+        <>
+          {element}
+          {focusVisible && !editable && (
+            <Role
+              children={keyboardDescription}
+              {...keyboardDescriptionProps}
+              id={keyboardDescriptionId}
+            />
+          )}
+        </>
+      ),
+      [
+        focusVisible,
+        editable,
+        keyboardDescription,
+        keyboardDescriptionProps,
+        keyboardDescriptionId,
+      ]
+    );
+
```

Comment:
Had to move down because otherwise the React fragment would be passed to the playground code component (to inject the class name) instead of the editor element.